### PR TITLE
Section 2.2 changes

### PR DESCRIPTION
**Pull request by jvdsn**
Changes justified inline.

### CODE REVIEW

File: cPP_DSC.adoc [hunk 1]
```diff
@@ -121,7 +121,7 @@ PP Date: {revdate}
 
 === TOE Overview
 
-The Target of Evaluation (TOE) is a Dedicated Security Component (DSC). In the context of this cPP, a DSC is the combination of a hardware component and its controlling OS or firmware. The firmware should be dedicated to providing the encompassing platform with services for the provisioning, protection, and use of Security Data Objects (SDOs), which include keys, identities, attributes, and other types of Security Data Elements (SDEs). See <<RepofTOE>> for an example of a TOE representation.
+The Target of Evaluation (TOE) is a Dedicated Security Component (DSC). In the context of this cPP, a DSC is the combination of one or more hardware component(s) and its controlling OS or firmware. The firmware should be dedicated to providing the encompassing platform with services for the provisioning, protection, and use of Security Data Objects (SDOs), which are composed of Security Data Elements (SDEs) such as keys, identities, attributes. See <<RepofTOE>> for an example of a TOE representation.
```
Comment (jvdsn) [inline cPP_DSC.adoc:124]: The paragraph below mentions that a DSC can contain multiple hardware components.

The sentence "use of Security Data Objects (SDOs), which include keys, identities, attributes, and other types of Security Data Elements (SDEs)." sounded like the SDOs are the keys/identities/attributes, but as I understand it those are the SDEs. The SDOs are composed of SDEs.

File: cPP_DSC.adoc [hunk 2]
```diff
@@ -175,7 +175,7 @@ image::TOEservices.png[,600]
 The labels in Figure 4 refer to the following:
 
 * SDE: Security Data Element
-* SDO: Security Data Object (component from SDEs and attributes)
+* SDO: Security Data Object (composed of SDEs and attributes)
```
Comment (jvdsn) [inline cPP_DSC.adoc:178]: I think this was simply a typo

File: cPP_DSC.adoc [hunk 3]
```diff
@@ -190,7 +190,7 @@ DSCs provide seven core security services to a platform as illustrated in <<Core
 |Description
 
 |Parse
-|The DSC shall ingest pre-provisioned keys, credentials, tokens, attributes, etc. from trusted components or services external to its boundary either across a secured channel or in a manner that the objects are protected for use only by the DSC.
+|The DSC shall ingest pre-installed keys, credentials, tokens, attributes, etc. from trusted components or services external to its boundary either across a secured channel or in a manner that the objects are protected for use only by the DSC.
```
Comment (jvdsn) [inline cPP_DSC.adoc:193]: This is one of the only two occurences of "pre-provisioned" in the document; everywhere else "pre-installed" is used. I suggest changing it here for consistency.

File: cPP_DSC.adoc [hunk 4]
```diff
@@ -237,7 +237,7 @@ Subjects: The following list contains the fundamental actors in the expected ope
 
 Objects: The following list contains objects the DSC expects to use during the expected operational use cases.
 
-* OB.P_SDO - Pre-provisioned SDOs (e.g. DSC.ID) with security attributes listed in the next paragraph.
+* OB.P_SDO - Pre-installed SDOs (e.g. DSC.ID) with security attributes listed in the next paragraph.
```
Comment (jvdsn) [inline cPP_DSC.adoc:240]: This is one of the only two occurences of "pre-provisioned" in the document; everywhere else "pre-installed" is used. I suggest changing it here for consistency.

File: cPP_DSC.adoc [hunk 5]
```diff
@@ -226,7 +226,7 @@ The security functional requirements rely on the following characteristics of th
...
+Users: The entities using the DSC will be client applications on the platform. They may be acting as proxies for users or may have identities of their own. The DSC will not be able to distinguish the difference; therefore, the cPP will recognize an entity known as the Client Application (CApp), as the user presenting authentication tokens and authorization values (collectively known as authorization data) to the DSC for the purposes of identity verification and authorization to perform operations. <<Roles>> discusses the concept of users in more detail. This cPP also recognizes a special user called the administrator, which typically has access to DSC objects normally denied to CApps (see definition of objects below).
```
Comment (jvdsn) [inline cPP_DSC.adoc:229]: Everywhere else, the acronym CApp is used, presumably to disambiguate it from the common Certificate Authority (CA) acronym. I suggest changing it here as well.

File: cPP_DSC.adoc [hunk 6]
```diff
@@ -249,7 +249,7 @@ Security Attributes: The following list contains the minimum security attributes
 
 * DSC.ID - The DSC identifier. It may also serve as the identifier for the DSC RoT.
 * CApp.ID - The Client Application identifier.
-* EPS.ID - The External Platform Storage (EPS) identifier. This attribute is optional for a passive EPS (i.e. plain memory that only stores information). If the DSC uses a Client Application to manage storage, then support for this attribute is required.
+* EPS.ID - The External Platform Storage (EPS) identifier. This attribute is optional for a passive EPS (i.e. plain memory that only stores information). If the DSC uses an active EPS to manage storage, then support for this attribute is required.
```
Comment (jvdsn) [inline cPP_DSC.adoc:252]: The term "active EPS" is used above.

**Comment by jvdsn** [comment]
I also have a few questions regarding this section.
* Section 2.2.1 mentions:
```
Pre-installed SDEs have two distinguishing characteristics:

* These keys may persist over a factory reset; and 
* They may not be accessible to administrators. 
```
The word "may" here seems to imply that these characteristics are not always applicable. Is that the intent?
* Table 1 mentions `from parsed or generated SDEs` for the Provision service. Are SDEs implicitly generated as part of this service? There doesn't seem to be any other conceptual service that does the generation.
* Section 2.2.4 distinguishes between pre-provisioned/pre-installed (OB.P_SDO) and transient SDOs (OB.T_SDO). What type of object would SDOs be when stored internally to the DSC, persistent across reboots, but not pre-installed by the manufacturer? Would they fall under OB.P_SDO or OB.T_SDO?
* Are all DSCs expected to assign all SDO.* attributes to all SDOs?
* Regarding OP.Import, the document says `The Admin may also give the DSC Authorization Data.` What does this refer to? Wouldn't all CApps be able to supply Authorization Data to the DSC, for authentication purposes? Or is this referring to default Authorization Data or Authorization Data for other Users?

File: cPP_DSC.adoc (hunk 1 above)
Comment (woodbe) [reply cPP_DSC.adoc:124]: I can agree with this change

**Review by woodbe — APPROVED** [review]

**Comment by slpotte** [comment]
Section 2.2.1 mentions:
Pre-installed SDEs have two distinguishing characteristics:

* These keys may persist over a factory reset; and 
* They may not be accessible to administrators. 
The word "may" here seems to imply that these characteristics are not always applicable. Is that the intent?
[SP: Yes, that is the intent.  There are some factory resets that intentionally erase important public firmware verification keys that intentionally brick the device.  But that is an unusual circumstance.]

Table 1 mentions from parsed or generated SDEs for the Provision service. Are SDEs implicitly generated as part of this service? There doesn't seem to be any other conceptual service that does the generation.
[SP: SDEs may be implicitly generated, but it's not required.]

Section 2.2.4 distinguishes between pre-provisioned/pre-installed (OB.P_SDO) and transient SDOs (OB.T_SDO). What type of object would SDOs be when stored internally to the DSC, persistent across reboots, but not pre-installed by the manufacturer? Would they fall under OB.P_SDO or OB.T_SDO?
[SP:  This sounds like they are persistent objects, so OB.P_SDO]

Are all DSCs expected to assign all SDO.* attributes to all SDOs?
[SP:  The SFRs are written that way.  The commercial vendors who were participating at the time didn't seem to have a problem with it, so we left it worded that way.]

Regarding OP.Import, the document says The Admin may also give the DSC Authorization Data. What does this refer to? Wouldn't all CApps be able to supply Authorization Data to the DSC, for authentication purposes? Or is this referring to default Authorization Data or Authorization Data for other Users?
[SP:  The first part of the paragraph refers to how the DSC receives objects from apps and external storage.  This phrase refers to what the DSC receives from Admins.  Inconsistent grammar.]

**Pull request merged**